### ISSUE TITLE
[A11y][SF][Table]: The chevron (>) elements column lack alternate text.

### ISSUE THREAD

**Issue by VidhiNPatel** [UI5/webcomponents#12657]
### Describe the bug

Chevron icons are used visually to indicate that the rows are interactive or actionable. This information should also be conveyed to users of screen readers. The cells in the last column receive focus when navigating with the arrow keys; therefore, they should have appropriate alternative text. 
Although the arrow icon SVGs now have aria-label="navigation", the presence of the aria-hidden attribute prevents them from being exposed to assistive technologies. Ensure that the name is exposed properly to screen readers.

Reference: https://ui5.sap.com/#/entity/sap.f.FlexibleColumnLayout/sample/sap.f.sample.FlexibleColumnLayoutWithOneColumnStart

### Reproduction steps

1. Go to https://ui5.github.io/webcomponents-react/v2/?path=/story/data-display-table--with-row-actions 
2. Ensure that the chevron (>) element aria-label name is exposed properly to screen readers.

### Expected Behaviour

Chevron icons are used visually to indicate that the rows are interactive or actionable. This information should also be conveyed to users of screen readers.
Reference: https://ui5.sap.com/#/entity/sap.f.FlexibleColumnLayout/sample/sap.f.sample.FlexibleColumnLayoutWithOneColumnStart

### UI5 Web Components for React Version

2.16

### UI5 Web Components Version

1.16

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by unazko** [comment]
Hello @UI5/ui5-team-tbl,

The reference scenario in sapui5 is different and there the navigation is represented via a gridcell whereas in the provided [sample](https://ui5.github.io/webcomponents-react/v2/?path=/story/data-display-table--with-row-actions) in ui5 webcomponents there are buttons in the last row cell. One of those buttons provides the details navigation action. 

When we navigate to the last cell with the three buttons all of the buttons are announced as "edit save navigation".
Couldn't test with JAWS.

Not sure how are they currently read out in the table as this the ui5-button doesn't currently expose public `accessibilityInfo` method, for which there is already a logged feature request:
https://github.com/UI5/webcomponents/issues/7729

Best regards,
Boyan
